addons/mail/static/src/core_models/global.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { many } from '@mail/model/model_field';
+
+registerModel({
+    name: 'Global',
+    lifecycleHooks: {
+        _created() {
+            odoo.__DEBUG__.messagingGlobal = this;
+        },
+        _willDelete() {
+            delete odoo.__DEBUG__.messagingGlobal;
+        },
```

Comment:
Just be aware that this is a problem in case there are 2 messaging services at once.
This is unfortunately the case with the website editor, as it's in an iframe.
It's already broken, so don't change anything in this PR: it's scope of another PR.